sections/discussion/category-topic.tex

```diff
@@ -1,8 +1,7 @@
 \subsection{Category-topic model}\label{sec:discussion_category_topic} 
-\vejleder[inline]{fix commas in this section}
 Specific observations for the category-topic model can also be made.
 Just as for the author-topic model, the similarity between pairs of categories can be calculated.
-Because topic distributions are generated for each category, just as they are for each author in the author-topic model, \autoref{eq:author_similarity} can be used for categories as well, with $i$ and $j$ being the categories.
+Because topic distributions are generated for each category, just as they are for each author in the author-topic model \autoref{eq:author_similarity} can be used for categories as well with $i$ and $j$ being the categories.
```

Comment:
```suggestion
Because topic distributions are generated for each category, just as they are for each author in the author-topic model, \autoref{eq:author_similarity} can be used for categories as well, with $i$ and $j$ being the categories.
```
Uden kommaerne bliver sætningen personligt svær at læse